cloudinit/util.py

```diff
@@ -1245,19 +1245,44 @@ def close_stdin():
         os.dup2(fp.fileno(), sys.stdin.fileno())
 
 
+def find_devs_with_freebsd(criteria=None, oformat='device',
+                           tag=None, no_cache=False, path=None):
+    if not criteria:
+        return glob.glob("/dev/msdosfs/*") + glob.glob("/dev/iso9660/*")
+    if criteria.startswith("LABEL="):
+        label = criteria.split("LABEL=")[1]
```

Comment:
Thanks for the change although I was trying to get at lstrip instead of split[<index>]
```suggestion
        label = criteria.lstrip("LABEL=")
```